File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo4/C/Exercicio5.c
/*5. Faça um programa que receba três números obrigatoriamente em ordem crescente e um quarto número que não siga essa regra. Mostre, em seguida, os quatro números em ordem decrescente. Suponha que o usuário digitará quatro números diferentes.*/

#include <stdio.h>

int main()
{
    float n1, n2, n3, n4;

    printf("Digite três números em ordem crescente: ");
    scanf("%f %f %f", &n1, &n2, &n3);
    printf("Digite um número aleatório: ");
    scanf("%f", &n4);

    if(n4 < n1){
        printf("%.1f -> %.1f -> %.1f -> %.1f\n", n3, n2, n1, n4);
    } else if(n4 < n2){
        printf("%.1f -> %.1f -> %.1f -> %.1f\n", n3, n2, n4, n1);
    } else if(n4 < n3){
        printf("%.1f -> %.1f -> %.1f -> %.1f\n", n3, n4, n2, n1);
    } else if(n4 > n3){
        printf("%.1f -> %.1f -> %.1f -> %.1f\n", n4, n3, n2, n1);
    }

    return 0;
}

File: Livro-FundamentosDaProgramacaoDeComputadores/Capitulo4/Python/Exercicio5.py
#5. Faça um programa que receba três números obrigatoriamente em ordem crescente e um quarto número que não siga essa regra.
# Mostre, em seguida, os quatro números em ordem decrescente. Suponha que o usuário digitará quatro números diferentes.*/

num1, num2, num3 = map(float,input("Digite três números em ordem crescente: ").split())
aleatorio = float(input("Digite um número fora da ordem anterior: "))

if aleatorio < num1:
    print("\n", num3, "-", num2, "-", num1, "-", aleatorio, "\n")
elif aleatorio < num2:
    print("\n", num3, "-", num2, "-", aleatorio, "-", num1, "\n")
elif aleatorio < num3:
    print("\n", num3, "-", aleatorio, "-", num2, "-", num1, "\n")
elif aleatorio > num3:
    print("\n", aleatorio, "-", num3, "-", num2, "-", num1, "\n")